### ISSUE TITLE
[BUG] agent status should be more verbose by default

### ISSUE THREAD

**Issue by dlouvier** [DataDog/datadog-agent#16818]
**Agent Environment**
`Agent 7.38.2 - Commit: ba442fd - Serialization version: v5.0.23 - Go version: go1.17.11`

**Describe what happened:**
`agent status` command print is insufficient when global logLevel is WARN or lower (less verbosity)

**Describe what you expected:**
`agent status` or any other command used for troubleshooting agent/check issues, should ignore global `logLevel` configuration and print by default enough information to help to troubleshoot.

We reduced decrease recently the default agent verbosity from default `INFO` to `WARN`, however, I was not expecting that when triggering the command to troubleshoot agent status, this will be taken into consideration. In addition, `agent status --help` doesn't show any possibility to increase the verbosity (e.g.: `-vvv`).

**Steps to reproduce the issue:**
1. Configure a `logLevel` to WARN or lower (less verbosity)
You can check current global logLevel with:
```
root@datadog-ldbsx:/# env | grep DD_LOG_LEVEL
DD_LOG_LEVEL=WARN
```

3. Run `agent status`
```
root@datadog-ldbsx:/# agent status
2023-04-28 07:56:35 UTC | CORE | WARN | (pkg/util/log/log.go:591 in func1) | Deactivating Autoconfig will disable most components. It's recommended to use autoconfig_exclude_features and autoconfig_include_features to activate/deactivate features selectively
2023-04-28 07:56:35 UTC | CORE | WARN | (pkg/secrets/secrets.go:50 in Init) | Agent configuration relax permissions constraint on the secret backend cmd, Group can read and exec
```
Only `WARN` level messages are printed.

4. Run `DD_LOG_LEVEL=INFO agent status`
It prints:
```
root@datadog-ldbsx:/# DD_LOG_LEVEL=INFO agent status | head -15
2023-04-28 07:56:53 UTC | CORE | WARN | (pkg/util/log/log.go:591 in func1) | Deactivating Autoconfig will disable most components. It's recommended to use autoconfig_exclude_features and autoconfig_include_features to activate/deactivate features selectively
2023-04-28 07:56:53 UTC | CORE | INFO | (cmd/system-probe/config/config.go:118 in Merge) | no config exists at /etc/datadog-agent/system-probe.yaml, ignoring...
2023-04-28 07:56:53 UTC | CORE | WARN | (pkg/secrets/secrets.go:50 in Init) | Agent configuration relax permissions constraint on the secret backend cmd, Group can read and exec
2023-04-28 07:56:53 UTC | CORE | INFO | (cmd/agent/app/status.go:125 in requestStatus) | Getting the status from the agent.
2023-04-28 07:56:55 UTC | CORE | INFO | (cmd/agent/app/status.go:163 in requestStatus) |
===============
Agent (v7.38.2)
===============

  Status date: 2023-04-28 07:56:53.355 UTC (1682668613355)
  Agent start: 2023-04-26 10:48:04.21 UTC (1682506084210)
  Pid: 96643
  Go Version: go1.17.11
  Python Version: 3.8.13
  Build arch: amd64
```

**Additional environment details (Operating System, Cloud provider, etc):**
Confirmed with GKE/Kubernetes Integration, probably happen with the other platforms.

**Comment by sgnn7** [comment]
Hi @dlouvier,
We discussed this within the team and as anticipated in my earlier comment we are generally leaning towards having a separately hard-coded level for `agent status` log output (`WARN`) with a configurable argument `--log-level` to decouple the main config from the CLI while still retaining consistency in expected UX.

We'll keep this ticket open and update it as we work on this.

**Comment by pgimalac** [comment]
Hello @dlouvier, I think we initially misunderstood your comment, the issue you report is that when setting `DD_LOG_LEVEL` to `warn`, the status isn't displayed anymore.
This is due to the status being displayed as a log, it was the case specifically in version 7.38 and it was fixed in 7.39.0 (see https://github.com/DataDog/datadog-agent/pull/13117), so this issue will be solved by updating to any more recent version of the `datadog-agent`.

Feel free to open the issue again if you have further comments.
